plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -435,7 +435,7 @@ func (n *NodeInfo) updateUsedPorts(pod *v1.Pod, used bool) {
 			// "0" is explicitly ignored in PodFitsHostPorts,
 			// which is the only function that uses this value.
 			if podPort.HostPort != 0 {
-				n.usedPorts[int(podPort.HostPort)] = used
+				n.usedPorts[fmt.Sprintf("%s %d %s", podPort.HostIP, podPort.HostPort, podPort.Protocol)] = used
```

Comment:
does it align with validation in API?

Reply:
see https://github.com/kubernetes/kubernetes/pull/39369. the order is a little different and it's using / as delimiter. Do we have to change it to align?

Reply:
prefer to make them align.

Reply:
agree，let me fix it